jr-extension-datetime/src/main/java/com/fasterxml/jackson/jr/extension/datetime/JacksonLocalDateTimeExtension.java

```diff
@@ -0,0 +1,11 @@
+package com.fasterxml.jackson.jr.extension.datetime;
+
+import com.fasterxml.jackson.jr.ob.JacksonJrExtension;
+import com.fasterxml.jackson.jr.ob.api.ExtensionContext;
+
+public class JacksonLocalDateTimeExtension extends JacksonJrExtension {
```

Comment:
As per my other note, should probably rename these NOT to refer to `LocalDateTime` since we probably do not want 12 different extensions for Java 8 added date/time types.

So something like `JacksonJavaTimeExtension`? And similarly for providers etc.

Whether extension should be `jr-extension-datetime` or `jr-extension-javatime` (latter just because JDK package is `java.time`) I leave up to you.

Reply:
yes that makes much more sense, I have changed the name of the module to `jr-extension-javatime`, and implemented the setter method required to set custom formatters:
![image](https://github.com/FasterXML/jackson-jr/assets/65479699/218add03-cd5a-443c-98b6-fb9bd336722e)
so now , if anyone wants to specify any custom one, it will just override the existing one, at the same time, providing with a 'sane default' choice, as before.